qtpy/tests/test_qtwidgets.py

```diff
@@ -1,9 +1,14 @@
 """Test QtWidgets."""
+import contextlib
 import sys
+from time import sleep
 
 import pytest
```

Comment:
```suggestion
import pytest
from pytestqt.exceptions import TimeoutError
```

`pytestqt` is a third-party import.